src/main/java/org/reaktivity/nukleus/route/RouteManager.java

```diff
@@ -41,4 +41,7 @@ MessageConsumer supplyReceiver(
     void setThrottle(
         long streamId,
         MessageConsumer throttle);
+
+    String resolveTag(
+        long routeId);
```

Comment:
Should not need any changes to the public API to resolve internal name mappings.